packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -129,6 +129,24 @@ export const start = async (zcf, privateArgs) => {
 
   /** @type {AddVaultType} */
   const addVaultType = async (collateralIssuer, collateralKeyword, rates) => {
+    const amountPattern = { brand: M.remotable(), value: M.any() };
+    const ratioPattern = {
+      numerator: amountPattern,
+      denominator: amountPattern,
+    };
```

Comment:
Should these be in some standard place?  Seems like we'll use them often.

Reply:
how about `packages/run-protocol/src/contractSupport.js`?

Reply:
That seems right.